databases/week3/querries.sql

```diff
@@ -0,0 +1,95 @@
+--Meals Querries
+--Get all meals
```

Comment:
You can also try to create the meal table first before getting them. Just in case.

Reply:
I have created the table and inserted the values in the table in the table-mealsharing.sql file.

Reply:
Cool if is the case, just ignore this comment.